include/wlr/types/wlr_seat.h

```diff
@@ -252,10 +252,25 @@ struct wlr_seat {
 		struct wl_signal request_set_cursor;
 
 		// wlr_seat_request_set_selection_event
```

Comment:
Can we move this comment to the right of `request_set_selection` and `set_selection`? This is a convention meaning the `data` listener argument is of this type.

Reply:
The comments are rather long, should i write them like this?

```c
	struct wl_signal request_set_selection; // lorem ipsum dolor sit amet
	                                        // this comment is indented with one tab and aligned
	                                        // with many spaces.
```

Reply:
I mean just the `// wlr_seat_request_set_selection_event` comment.

Reply:
Or just add a plain-text sentence: "The signal data is a pointer to a struct wlr_seat_request_set_selection_event."